charts/aws-ebs-csi-driver/templates/_helpers.tpl

```diff
@@ -41,8 +41,12 @@ helm.sh/chart: {{ include "aws-ebs-csi-driver.chart" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
+app.kubernetes.io/component: csi-driver
```

Comment:
I am not familiar enough with csi, in general, to know if `csi-driver` is a sufficient component name or if it should be `ebs-csi-driver`.  @wongma7 @ayberk thoughts?

Reply:
I think it's OK, the example from https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/ is `database` which is very generic. https://github.com/kubernetes-csi/csi-driver-host-path/blob/master/deploy/kubernetes-1.21/hostpath/csi-hostpath-driverinfo.yaml#L9 also is setting it so lgtm